docs/api/overview.md

```diff
@@ -1,57 +1,79 @@
+
+Vega is a public, decentralised network run by independent validators. 
+
+There are a number of different servers that you'll need to interact while working with the network - data will generally be recieved from one service and transactions will be sent to another using data nodes. See the [data flow section on the architecture page](./architecture.md#data-flow).
+
+:::caution Supplying your own data
+Anyone that plans to build significant integrations, trading bots, or power backed services should expect to run their own data node, or collaborate with a data node provider, if one exists. 
+:::
+
+## Authentication
+Authentication takes several forms when interacting with Vega. When requesting data from a standard data node, such as through a query, you don't need to authenticate nor do you need an API token. Service providers offering free or paid data nodes may require you to authenticate, however. If this is the case you will need to refer to the provider's documentation for details of how to do so.
+
+If you want to submit transactions to the network, you'll need to ensure that you have a Vega Wallet, which provides authentication in the form of a signature when sending transactions. 
+
+When creating any scripts or software to interact with Vega, you'll need a wallet authentication token. 
+
+:::note Go deeper
+**[Wallet API guides](./vega-wallet/before-you-start.md)**: See the wallet API documentation for more on authentication tokens.
+:::
 
-As most of the APIs are designed to use for trading-related queries, the best place to try them out is on the testnet network, also known as Fairground. 
+## Sending transactions to the chain
+When sending transactions, you'll need a Vega Wallet with at least one keypair, and have the wallet service running. You'll need to have your Vega public key (also known as a party ID) to hand, as well as the relevant transaction data, in order to submit your transaction.
 
-## Tutorials
-Tutorials provide the information you'll need about the protocol to understand and use the guide, as well as instructions on how to interact with scripts, API calls, or other code. 
+To access the Vega network, the wallet needs to be configured with the location of one of more [data nodes](./architecture.md#data-nodes).
 
-**[Tutorials](../tutorials)**: See the tutorial(s) currently available for this network.
+When your client wants to send a transaction using the Vega wallet API, it will construct the transaction in JSON and pass it to the wallet. The wallet performs a client-side proof-of-work calculation, signs the transaction and forwards it to a validator node before it is added to a block. It is also possible to have the wallet sign a transaction without sending it, if needed. Alternatively, you can build the signer into your client, though you'll need to account for the PoW calculations.
```

Comment:
It's said a couple of other times here, but the transaction is not specifically sent to a validator node. It's sent to a node on the network, which implement the CoreService grpc service. This is implemented by either a datanode api, core full node api or core validator.